docs/Customs Declare Exports AutoComplete.js

```diff
@@ -793,19 +797,22 @@ function additionalInformation(){
     }
 }
 
-function addDocuments(){
-    if (currentPageIs('/customs-declare-exports/declaration/items/.*/additional-documentation-list')) {
+function additionalInformationSummary(){
+    if (currentPageIs('/customs-declare-exports/declaration/items/.*/additional-information-list')) {
         document.getElementById('code_no').checked = 'checked';
-        document.getElementById('submit').click();
+        document.getElementById('submit').click()
     }
+}
+
+function addDocuments(){
     if (currentPageIs('/customs-declare-exports/declaration/items/.*/additional-documentation')) {
         switch(getDeclaration())
         {
             case 'C':
                 if (!document.querySelector("#content>article>form>table>tbody>tr>th")) {
                     document.getElementById('documentTypeCode').value ='C601';
                     document.getElementById('documentIdentifier').value ='GBIPO717572504502801';
-                    document.getElementById('add').click();
+                    document.getElementById('submit').click();
```

Comment:
Has the button id changed on these pages?  Or did this never work?

Reply:
it was not working - it needs a biggest changes if we want to add more then one thing